README.md

```diff
@@ -1,2 +1,131 @@
-# bfi-iiif-logging
-Solution for BFI National Archive Universal Viewer deployment, to log users in, track their interactions with the IIIF resources in UV, and output to a log.
+# BFI - IIIF Universal Viewer Logging Platform
+
+## Introduction
+
+This project contains an implementation of the
+[Universal Viewer](https://universalviewer.io/) served by a Kotlin
+backend, which logs interactions with the Universal Viewer (and thus,
+the underlying IIIF manifests and images) in a MySQL database for long
+term auditing and analytics.
+
+This project is delivered as part of the British Film Institute's wider
+Data and Digital Preservation platform.
+
+The project is built using
+[GitHub Actions](https://github.com/bfidatadigipres/bfi-iiif-load-balancer/actions),
+and the produced containers are persisted in
+[GitHub Container Registry](https://github.com/orgs/bfidatadigipres/packages/container/package/bfi-iiif-load-balancer).
+
+## Build
+
+[![build](https://github.com/bfidatadigipres/bfi-iiif-logging/actions/workflows/build.yml/badge.svg)](https://github.com/bfidatadigipres/bfi-iiif-logging/actions/workflows/build.yml)
+
+The project uses Kotlin and is built using Gradle:
+
+```bash
+./gradlew clean build
+```
+
+The path [`src/main/resources/static`](src/main/resources/static) is a
+Git Submodule, pointing to a repository containing a configuration of
+the Universal Viewer (i.e. an `index.html` and associated
+`uv-config.json`, `manifests.json`, etc). During the build process, the
+Universal Viewer bundle is downloaded and unpacked into this directory.
+
+A `Dockerfile` is provided which builds and exposes the application on
+port `8080`.
+
+A `docker-compose.dev.yml` manifest is provided for running the
+application on a local development machine. The manifest will build,
+run, and expose the application:
+
+```bash
+docker-compose --file docker-compose.dev.yml up --build --remove-orphans
+```
+
+## Deployment
+
+### Installation
+
+The [`deploy/`](deploy/) directory contains a directory hierarchy and
+configuration files required for a deployment to a Linux environment,
+scoped to a specific environment (e.g. `DEV`, `UAT`, `PROD`, etc):
+
+- `/etc/`
+  - `/opt/bfi/iiif-logging/<environment>/`
```

Comment:
Something I hadn't noticed before, using `<` and `>` in paths makes a git checkout fail on Windows. Might be worth renaming this directory.